languages/en.php

```diff
@@ -154,6 +154,7 @@
 	'access:write' => "Write access",
 	'access:admin_only' => "Administrators only",
 	'access:missing_name' => "Missing access level name",
+	'access:comments:change' => "There are comments on this content, by changing the access level you will also be changing the access of those comments.  Do you want to continue?",
```

Comment:
I guess we can remove it if it's public, but that doesn't change the point and makes it less consistent.  It could also change from ACCESS_LOGGED_IN to a friend collection.  I could easily determine whether it's being more or less restrictive for the baked in access of private/friends/logged in/public but when we start considering group acls and custom acls it gets tricky.  The only thing I can think of to resolve those is an ajax callback that compares the acl membership, and that seems a bit overkill.